droolsjbpm-integration-tests/src/test/java/org/drools/droolsjbpm/integrations/tests/MBeansMonitoringWithJBpmTest.java

```diff
@@ -0,0 +1,366 @@
+
+    @After
+    public void tearDown()
+            throws Exception {
+    	if (mbeansprop != null) {
+    		System.setProperty( MBeansOption.PROPERTY_NAME, mbeansprop );
+    	} else {
+    		System.setProperty( MBeansOption.PROPERTY_NAME, MBeansOption.DISABLED.toString() );
+    	}
+    }
+    
+    @Test
+    public void testRulesAndProcesses() throws Exception {
+        String drl = "package org.drools.droolsjbpm_integration_testmgt\n" +
+    
+                "rule S\n" +
+                "when\n" +
+                "    $s: String() \n" +
+                "then\n" +
+                "    try { Thread.sleep(10); } catch (Exception e) { }\n" + 
+                "    System.out.println($s); \n" +
+                "end\n"
+                
+                ;
+        
+        System.out.println(drl);
+        
+        String process = "<?xml version=\"1.0\" encoding=\"UTF-8\"?> \n" + 
+                "<definitions id=\"Definition\"\n" + 
+                "             targetNamespace=\"http://www.example.org/MinimalExample\"\n" + 
+                "             typeLanguage=\"http://www.java.com/javaTypes\"\n" + 
+                "             expressionLanguage=\"http://www.mvel.org/2.0\"\n" + 
+                "             xmlns=\"http://www.omg.org/spec/BPMN/20100524/MODEL\"\n" + 
+                "             xmlns:xs=\"http://www.w3.org/2001/XMLSchema-instance\"\n" + 
+                "             xs:schemaLocation=\"http://www.omg.org/spec/BPMN/20100524/MODEL BPMN20.xsd\"\n" + 
+                "             xmlns:bpmndi=\"http://www.omg.org/spec/BPMN/20100524/DI\"\n" + 
+                "             xmlns:dc=\"http://www.omg.org/spec/DD/20100524/DC\"\n" + 
+                "             xmlns:di=\"http://www.omg.org/spec/DD/20100524/DI\"\n" + 
+                "             xmlns:tns=\"http://www.jboss.org/drools\">\n" + 
+                "\n" + 
+                "  <process processType=\"Private\" isExecutable=\"true\" id=\"com.sample.HelloWorld\" name=\"Hello World\" >\n" + 
+                "\n" + 
+                "    <!-- nodes -->\n" + 
+                "    <startEvent id=\"_a1\" name=\"StartProcess\" />\n" + 
+                "    <scriptTask id=\"_a2\" name=\"Hello\" >\n" + 
+                "      <script>System.out.println(\"Hello World\");</script>\n" + 
+                "    </scriptTask>\n" + 
+                "    <endEvent id=\"_a3\" name=\"EndProcess\" >\n" + 
+                "        <terminateEventDefinition/>\n" + 
+                "    </endEvent>\n" + 
+                "\n" + 
+                "    <!-- connections -->\n" + 
+                "    <sequenceFlow id=\"_a1-_a2\" sourceRef=\"_a1\" targetRef=\"_a2\" />\n" + 
+                "    <sequenceFlow id=\"_a2-_a3\" sourceRef=\"_a2\" targetRef=\"_a3\" />\n" + 
+                "\n" + 
+                "  </process>"+
+                "  <process processType=\"Private\" isExecutable=\"true\" id=\"com.sample.CiaoWorld\" name=\"Ciao World\" >\n" + 
```

Comment:
I think this is the first time I see use of multiple processes defined in single definitions :) nothing wrong with that as it seems to work fine